### ISSUE TITLE
Unable to check out specific commits

### ISSUE THREAD

**Issue by kelvSYC-ticketfly** [jfrog/bower-art-resolver#23]
Checking out specific commits from artifactory results in a 404:

GET my.artifactory.repo/artifactory/api/bower/my-bower-remote/binaries/alex-seville/blanket.git/5e94fc30f2e694bb5c3718ddcbf60d467f4b4d26 returns

```
{
  "errors" : [ {
    "status" : 404,
    "message" : "No version found for 5e94fc30f2e694bb5c3718ddcbf60d467f4b4d26"
  } ]
}
```

It seems that only branches and tags are supported, and due to the 404, it will not delegate to another resolver that could successfully retrieve the artifact; some assistance would be appreciated.

**Comment by eyalbe4** [comment]
@kelvSYC-ticketfly, you are correct. Artifactory does not yet support the download of a specific commit. We opened [RTFACT-12424](https://www.jfrog.com/jira/browse/RTFACT-12424) to add this functionality. Please watch this Jira issue for updates.
